### PR TITLE
Fix _meta warnings on django1.8

### PR DESCRIPTION
**Pull request by collinanderson**

### CODE REVIEW

**Comment by kaedroho** [comment]
Shall I merge this commit into #297?

**Comment by frewsxcv** [comment]
What's the status of this PR? How does it relate to #297?

**Comment by collinanderson** [comment]
The last commit here fixes a few deprecation warnings in 1.8. We could either merge my commit into #297, or I could rebase it once #297 is merged.

**Comment by frewsxcv** [comment]
@collinanderson Can you rebase off the current `develop` branch since #297 has merged?

**Comment by collinanderson** [comment]
done

File: taggit/managers.py [hunk 1]
```diff
@@ -49,6 +49,13 @@ def _model_name(model):
         return model._meta.model_name
 
 
+def _get_field(model, name):
```
Comment (frewsxcv) [inline taggit/managers.py]: Can you move this to `taggit/utils.py`?

File: taggit/models.py [hunk 2]
```diff
@@ -101,11 +102,15 @@ class Meta:
 
     @classmethod
     def tag_model(cls):
-        return cls._meta.get_field_by_name("tag")[0].rel.to
+        if VERSION < (1, 8):
+            return cls._meta.get_field_by_name('tag')[0].rel.to
+        return cls._meta.get_field('tag').rel.to
```
Comment (frewsxcv) [inline taggit/models.py]: Once the other function gets moved to `utils.py`, can you utilize it here so we can reduce the number of places we do version checking

File: taggit/managers.py (hunk 1 above)
Comment (collinanderson) [inline taggit/managers.py]: done

File: taggit/models.py (hunk 2 above)
Comment (collinanderson) [inline taggit/models.py]: done

**Comment by frewsxcv** [comment]
Thanks for this @collinanderson!

**Comment by frewsxcv** [comment]
I just released `0.14.0` that includes this change

https://pypi.python.org/pypi/django-taggit/0.14.0

**Pull request merged**